client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -173,11 +179,13 @@ class TransferDomainPrecheck extends React.Component {
 			);
 		}
 
-		if ( loading && ! isStepFinished ) {
+		if ( loading && ! isStepFinished && ! unlockCheckClicked ) {
 			message = translate( 'Please wait while we check the lock staus of your domain.' );
 		}
 
-		const buttonText = translate( "I've unlocked my domain" );
+		const buttonText = unlockCheckClicked
+			? translate( 'Check again' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Check Again' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).